metadata-service/auth-servlet-impl/src/main/java/com/datahub/auth/authentication/AuthServiceController.java

```diff
@@ -193,7 +193,7 @@ CompletableFuture<ResponseEntity<String>> signUp(final HttpEntity<String> httpEn
     try {
       bodyJson = mapper.readTree(jsonStr);
     } catch (JsonProcessingException e) {
-      log.debug("Failed to parse json while attempting to create native user", e);
+      log.error("Failed to parse json while attempting to create native user ", e);
```

Comment:
why is this adding an space character at the end? it feels like this is some bad automation